hugo.toml

```diff
@@ -44,15 +46,15 @@ name = "linkedin"
 url = "https://linkedin.com/pavananarayanak"
 
 [languages.en.params.portrait]
-path = "/img/avatar.jpg"
+path = "/img/avatar.png"
```

Comment:
Will have to update favicons and other image of size 16x16, 32x32 and 256x256 as well depending on the requirements.